lifecycle-app/src/App.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import "./App.css";
+import Form from "./components/Form";
+import ToDoList from "./components/ToDoList";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+class App extends React.Component {
+  state = {
+    input: "",
+    toDoList: []
+  };
+
+  notify = input => {
```

Comment:
I dont think you would need this function, you can just directly use the toast component